packages/teams-js/src/public/constants.ts

```diff
@@ -52,6 +52,11 @@ export enum HostName {
    */
   orange = 'Orange',
 
+  /**
+   * Microsoft connected workplace platform
+   */
+  places = 'Places',
```

Comment:
oh, is "places" ready to be added as enum? (not sure what stage they are currently in).
If so, shall we add the same thing in mobile as well?